sig-windows/charter.md

```diff
@@ -14,13 +14,14 @@ Windows specific implementation.
 
 #### Code, Binaries and Services
 
-- Windows specific code in all parts of the codebase.
+- Windows specific code in all parts of the codebase
 - Testing of Windows specific features and clusters
 
 #### Cross-cutting and Externally Facing Processes
 
 - Work with other SIGs on areas where Windows and Linux (and possibly other OSes in the future) deviate from one another in terms of functionality.
-
+- Providing community guidance for external tools and paradigms necessary for production windows k8s administration
+- Helping to transition from CSI/CNI tooling workarounds to the emerging (windows privileged containers)[https://github.com/ambguo/enhancements/blob/master/keps/sig-windows/1981-windows-privileged-container-support/20201981-windows-privileged-container-support.md] initiative
```

Comment:
i would remove that part right now. privileged containers is unproven yet and we have not made any decisions on long term impact.

Reply:
hmm, ok, i guess part of my goal was to actually get us to actively own the runtime story, maybe thats not in the interests of the sig in general?  thats fine if thats the case - maybe we should be explicit about that?

Reply:
^ might be worthy of creating a "Windows container runtime" working group  ....... under the braoder sig ?

Reply:
updatedd to focus on reference application rather then wins/privileged containers